DataFormats/Detectors/TRD/include/DataFormatsTRD/CalT0.h

```diff
@@ -32,11 +32,34 @@ class CalT0
   ~CalT0() = default;
 
   void setT0(int iDet, float t0) { mT0[iDet] = t0; }
+  void setT0av(float t0) { mT0av = t0; }
 
   float getT0(int iDet) const { return mT0[iDet]; }
+  float getT0av() const { return mT0av; }
+  float calcT0av() const
```

Comment:
here I was a bit confused first that you have in principle two getters for the average T0. Could you please add some comment(s) that mT0av is the average value that you get when you take the PH data from all chambers combined and fit that and `calcT0av` returns the average T0 from all individual detectors?

Reply:
added the comments

Reply:
added the comments